core/build.gradle

```diff
@@ -65,6 +65,9 @@ dependencies {
     compile (group: 'com.google.oauth-client',      name: 'google-oauth-client-jetty',      version: '1.17.0-rc')   { exclude module: 'servlet-api' }
     compile ("com.sequenceiq:azure-rest-client:${azureRestClientVersion}")                                          { exclude group: 'log4j' }
     compile (group: 'com.amazonaws',                name: 'aws-java-sdk',                   version: '1.9.35')      { exclude group: 'commons-logging' }
+    compile group: 'com.jcraft',                    name: 'jsch',                           version: '0.1.52'
+    compile group: 'com.hierynomus',                name: 'sshj',                           version: '0.11.0'
```

Comment:
sshj is much simpler and easier to use, e.g.: it doesn't need a known_hosts file to validate the host,  it is as simple as this: `ssh.addHostKeyVerifier(sshFingerprint);`, but it cannot generate ssh keypairs, jsch is used only for that.